### PR TITLE
chore(sdk-core): Refactor & split getVerifyAndSignBitGoChallenges

### PR DESCRIPTION
**Pull request by mohammadalfaiyazbitgo**
- Add method to sign bitgo challenge without verification. To be used in contexts where the caller will verify challenges/proofs separately from signing.

### CODE REVIEW

**Review by zahin-mohammad — DISMISSED** [review]
lgtm. Existing function was split up, so the existing tests still cover these changes.

File: package.json [hunk 1]
```diff
@@ -69,7 +69,7 @@
     "parse-path": "^5.0.0",
     "parse-url": "8.1.0",
     "terser": "^5.14.2",
-    "vm2": "^3.9.17",
```
Comment (pranavjain97) [inline package.json:72]: was this intentional?

Comment (mmcshinsky-bitgo) [reply package.json:72]: Security audit update.

**Review by zahin-mohammad — DISMISSED** [review]

File: package.json (hunk 1 above)
Comment (zahin-mohammad) [reply package.json:72]: https://bitgo.slack.com/archives/C010AEXLLCR/p1684247091685949?thread_ts=1684246605.689579&cid=C010AEXLLCR

Comment (zahin-mohammad) [reply package.json:72]: @mohammadalfaiyazbitgo shouldn't there be a package-lock change as well with this? Did you run `yarn`?

Comment (mohammadalfaiyazbitgo) [reply package.json:72]: yeah forgot that, I'll update

**Review by zahin-mohammad — APPROVED** [review]

**Pull request merged**